samples/switchless/host/host.c

```diff
@@ -82,8 +82,40 @@ int main(int argc, const char* argv[])
         flags |= OE_ENCLAVE_FLAG_SIMULATE;
     }
 
-    // Enable switchless and configure host worker number
-    oe_enclave_setting_context_switchless_t switchless_setting = {1, 0};
+    // To demonstrate the benefit of using switchless ocalls, we will
+    // first make 1000000 switchless ocalls from the main thread and measure
+    // the time taken. Then we will make the same number of regular ocalls
+    // and measure the time taken. Using the two measurements, we can arrive at
+    // the speedup factor that switchless ocalls provide over normal ocalls.
+    //
+    // Similarly, to demonstrate the benefit of using switchless ecalls, we will
+    // first make 1000000 switchless ecalls from the main thread and measure the
+    // time taken. Then we will make the same number of regular ecalls and
+    // measure the time taken. Using the two measurements, we can arrive at the
+    // speedup factor that switchless ecalls provide over normall ecalls.
+    //
+    // We will conservatively assume that we are running on a two core machine.
```

Comment:
I think the first sentence should say something like:
"We conservatively assume that we are executing on a machine with 1 available core."  (i.e., 1 not 2, to be conservative)

Reply:
I have rephrased it to..

"In this sample, the main thread will first keep posting switchless ocalls from within the enclave. To service these ocalls, .... One a machine with atleast two cores, both the caller and worker threads can potentially run uninterrupted on dedicated cores, causing less contention and leading to maximum speedup via switchless calling. On a machine with more than two cores, ..."